ipfs-api/Cargo.toml

```diff
@@ -28,5 +28,19 @@ ipfs-api-backend-hyper    = { version = "0.5", path = "../ipfs-api-backend-hyper
 
 [dev-dependencies]
 actix-rt                  = "2.5"
+cfg-if                    = "1"
 futures                   = "0.3"
-tokio                     = { version = "1", features = ["rt-multi-thread", "macros"] }
+ipfs-api-versions         = { version = "0.1", path = "../ipfs-api-versions" }
+passivized_docker_engine_client = "0.0.5"
```

Comment:
Just FYI, I'm having some trouble running my tests locally on Fedora 36:

```
DockerClientUse(UnparseableJsonResponse { status: 200, text: "...", parse_error: Error("invalid type: null, expected a map", line: 1, column: 2777) })', ipfs-api/tests/test_basic_auth.rs:28:10
```

Would like to get this working locally before merging!

Reply:
I was able to reproduce this in the underlying library on Fedora 37 and fixed it.

Reply:
Works now!